package-lock.json

```diff
@@ -3170,7 +3170,7 @@
     },
     "callsites": {
       "version": "2.0.0",
-      "resolved": "https://registry.npmjs.org/callsites/-/callsites-2.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/callsites/-/callsites-2.0.0.tgz",
```

Comment:
That has nothing to do with the story and looks a bit dangerous - please remove.

Reply:
done